rebasehelper/config.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os.path
+
+from six.moves import configparser
+
+from rebasehelper.logger import logger
+from rebasehelper.options import OPTIONS, traverse_options
+from rebasehelper.constants import DEFAULT_CONFIG_LOCATIONS
+
+
+class Conf:
+    def __init__(self, config_file):
+        self.path_to_config = self.get_conf_path(config_file)
+        self.config = self.get_config()
+
+    @staticmethod
+    def get_conf_path(config_file):
+        if not config_file:
+            path = os.environ.get(DEFAULT_CONFIG_LOCATIONS[0], os.path.expandvars(DEFAULT_CONFIG_LOCATIONS[1]))
```

Comment:
This is not what I had in mind. Defining this as constants is unnecessary and actually much less readable. Also those two constants do not have the same kind of content, the first one is name of an environment variable while the second one is a path.